docs/content/users/quickstart.md

```diff
@@ -366,14 +366,14 @@ Start a new [Kirby CMS](https://getkirby.com) project or use an existing one.
 
 Use a new or existing Composer project, or clone a Git repository.
 
-The Laravel project type can be used for [Lumen](https://lumen.laravel.com/) like it can for Laravel. DDEV automatically updates or creates the `.env` file with the database information.
+The Laravel project type can be used for [StarterKits](https://laravel.com/docs/11.x/starter-kits), [Laravel Livewire](https://livewire.laravel.com/) and others - like it can for Laravel. DDEV automatically updates or creates the `.env` file with the database information.
```

Comment:
I think we can remove the first sentence entirely, because Starter Kits are extensions of Laravel, and the base for all of them is `laravel/framework`, so it's not correct to compare them with Laravel. The situation was different with Lumen.

Reply:
```suggestion
The Laravel project type can be used for [StarterKits](https://laravel.com/docs/11.x/starter-kits), [Laravel Livewire](https://livewire.laravel.com/) and others, just as it is used with Laravel. DDEV automatically updates or creates the `.env` file with the database information.
```

Reply:
@rfay yes, that sounds better - I made a small change for your comment, removed the `11.x` from the URL.